.circleci/scripts/git-diff-develop.ts

```diff
@@ -6,6 +6,26 @@ import { promisify } from 'util';
 
 const exec = promisify(execCallback);
 
+const MAIN_BRANCH = 'develop';
+
+/**
+ * Get the target branch for the given pull request.
+ *
+ * @returns The name of the branch targeted by the PR.
+ */
+async function getBaseRef(): Promise<string | null> {
+  if (!process.env.CIRCLE_PULL_REQUEST) {
+    return null;
+  }
+
+  // We're referencing the CIRCLE_PULL_REQUEST environment variable within the script rather than
+  // passing it in because this makes it easier to use Bash parameter expansion to extract the
+  // PR number from the URL.
+  const result = await exec(`gh pr view --json baseRefName "\${CIRCLE_PULL_REQUEST##*/}" --jq '.baseRefName'`);
```

Comment:
I tried using the `CIRCLE_PR_NUMBER` environment variable first, but it doesn't appear to be present for our workflows for some reason, despite being documented.

Sadly there is no way to reference the base ref directly from CircleCI either. It's a highly requested feature apparently, but is not supported. But this works fine as a workaround.